src/devtools/views/events/module.js

```diff
@@ -24,6 +24,8 @@ const mutations = {
     state.inspectedIndex = -1
   },
   'INSPECT' (state, index) {
+    if (index < 0) index = 0
+    if (index >= state.events.length) index = state.events.length - 1
```

Comment:
wouldn't this be enough actually?
```
state.inspectedIndex = Math.max(0, state.events.length - 1)
```

Reply:
No? 🙃

Reply:
Ah, true. It would have to be:
```
Math.min(Math.max(0, index), state.events.length - 1)
```
but it wouldn't be necessarily more readable ;D